### PR TITLE
Switch to cdylib instead of binary

### PR DESCRIPTION
**Pull request by badboy**

### CODE REVIEW

File: src/compile.js [hunk 1]
```diff
@@ -51,34 +51,29 @@ async function rustToWasm(rustSource, dependencies = {}) {
...
-      path = "main.rs"
+      [lib]
+      path = "lib.rs"
+      crate-type = ["cdylib"]
 
       [dependencies]
       ${dependenciesToml}
-
-      [profile.release]
-      opt-level = "s"
```
Comment (killercup) [inline src/compile.js]: y u no keep file small

**Review by killercup — COMMENTED** [review]
Awesome! Just on es small bit then this is good to go!

File: src/compile.js (hunk 1 above)
Comment (badboy) [reply src/compile.js]: oops

**Review by killercup — APPROVED** [review]
:shipit: 

bors r+

**Pull request merged**